package.json

```diff
@@ -379,7 +378,6 @@
     },
     "resolutions": {
         "ip": "1.1.8",
-        "terser-webpack-plugin": "^4.2.3",
```

Comment:
removing this means that versions less than 4.x can be used and I assume this was added to resolve some security bug. if you check the `yarn.lock` version `1.4.5` is now installed. what's the reason for removing?

Reply:
This is removed because of the chromatic pipeline error - https://app.circleci.com/pipelines/github/box/box-ui-elements/17861/workflows/f30ce332-33e8-4e0b-81f6-1c3b7dd4de9e/jobs/50397?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary

Reply:
This was forcing a downgrade which threw errors in webpack 5 when it comes with v5